### PR TITLE
EVAKA-3831 Create ghost unit flag

### PR DESCRIPTION
**Pull request by mapehe**
#### Checklist for pull request creator

- [ ] The code is consistent with the existing code base
- [ ] Tests have been written for the change (e2e, integration tests)
- [ ] The change has been tested locally
- [ ] The change conforms to the UX specifications
- [ ] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [ ] The branch has been rebased against master before the PR was created

#### Checklist for pull request reviewer (copy to review text box)

```
- [ ] All changes in all changed files have been reviewed
- [ ] The code is consistent with the existing code base
- [ ] Tests have been written for the change (e2e, integration tests)
- [ ] The change has been tested locally
- [ ] The change conforms to the UX specifications
- [ ] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [ ] The PR branch has been rebased against master and force pushed if necessary before merging
```

### CODE REVIEW

**Comment by mapehe** [comment]
Noticed that `roundTheClock` flag was [previously](https://github.com/espoon-voltti/evaka/commit/e0f46c4adae4e4ab9a6a077cd191efc22ad2a410) removed. There is probably a reason why it was not removed from the frontend form, but decided to point it out as a double check.

**Comment by mapehe** [comment]
Also `ghostUnit` sounds a bit like a bad action movie, but couldn't come up with a better translation.

**Comment by mapehe** [comment]
Added an e2e test for when an admin submits the unit details form.

File: frontend/e2e-test/test/e2e/dev-api/fixtures.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export const daycareFixture: Daycare = {
   id: '4f3a32f5-d1bd-4b8b-aa4e-4fd78b18354b',
   careAreaId: careAreaFixture.id,
   name: 'Alkuräjähdyksen päiväkoti',
-  type: ['CENTRE'],
+  type: ['CENTRE', 'PRESCHOOL'],
```
Comment (mapehe) [inline frontend/e2e-test/test/e2e/dev-api/fixtures.ts:82]: Not sure if this is a great idea, but it avoids [this](https://github.com/espoon-voltti/evaka/blob/b88fc70c7c767211bd918fb40779ed60dc7edfb0/frontend/packages/employee-frontend/src/components/unit/unit-details/UnitEditor.tsx#L331) error.

**Review by reynders — APPROVED** [review]
Good stuff!!

File: frontend/e2e-test/test/e2e/specs/5_employee/unit-editor.spec.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  })
+  .afterEach(logConsoleMessages)
+  .after(async () => {
+    await cleanUp()
+  })
+
+test('Admin can edit unit details', async () => {
+  await unitDetailsPage.openUnitDetailsPageById(daycare1.id)
+  await unitDetailsPage.enableUnitEditor()
+  await unitDetailsPage.fillManagerDataAndSubmitForm()
```
Comment (Joosakur) [inline frontend/e2e-test/test/e2e/specs/5_employee/unit-editor.spec.ts:40]: Here you could also do something like
```
await unitDetailsPage.fillManagerData()
await unitDetailsPage.submitForm()
// todo: verify that the updated manager data is visible somewhere
```

File: frontend/e2e-test/test/e2e/dev-api/fixtures.ts (hunk 1 above)
Comment (Joosakur) [reply frontend/e2e-test/test/e2e/dev-api/fixtures.ts:82]: This is fine 👍

**Review by Joosakur — APPROVED** [review]
Great work! 💎

**Review by hruoho — APPROVED** [review]
Looks great! :medal_sports:

File: frontend/e2e-test/test/e2e/specs/5_employee/unit-editor.spec.ts (hunk 2 above)
Comment (mapehe) [reply frontend/e2e-test/test/e2e/specs/5_employee/unit-editor.spec.ts]: Thanks. Agreed, refactored accordingly in f2047b6

**Pull request merged**